content/usgs-packages/geoknife_Assessment.Rmd

```diff
@@ -46,4 +47,70 @@ Before starting the exercises, you should make sure that the `geoknife` package
 library(geoknife)
 ```
 
+## Exercise 1
 
+*How many GDP data sets are related to sea level rise? Hint: `grep`.*
+
+```{r addToggle='ON'}
+# First, you need to query for all web data
+all_webdata <- query("webdata")
+all_titles <- title(all_webdata)
+all_abstracts <- abstract(all_webdata)
+
+# Then start sleuthing using `grep`
+keyword_str <- "sea level rise|sea level|sea level|sea rise"
+index_t <- grep(keyword_str, all_titles, ignore.case = TRUE)
+index_a <- grep(keyword_str, all_abstracts, ignore.case = TRUE)
+index_both <- unique(c(index_t, index_a))
+
+# Look at the titles and abstracts of datasets that match your criteria
+length(index_both)
+all_titles[index_both]
+all_abstracts[index_both]
+```
+
+## Exercise 2
+
+*What variables are in the "University of Idaho Daily Meteorological data for continental US" dataset? Also, choose one variable and determine the range of dates. Hint: create a geoknife fabric first.*
+
+```{r addToggle='ON'}
+# Use the all_webdata object from Exercise 1 to create the fabric
+us_meterology <- webdata(all_webdata["University of Idaho Daily Meteorological data for continental US"])
```

Comment:
You might want to make the exercises independent, which while forcing you to redo the work keeps it clear what is going on.  Carrying over `all_webdata` from Excercise 1 might be confusing if the student skips over that exercise in some way.  Just a thought.